ozaria/site/components/teacher-dashboard/BaseTeacherLicenses/common/LicenseCard.vue

```diff
@@ -190,6 +202,13 @@ export default {
   font-weight: 600;
 }
 
+.special {
+    font-size: 12px;
```

Comment:
is indentation here more than .dates section 🤔

Reply:
ha cool. eslint doesn't fix style indentation.